Makefile

```diff
@@ -42,6 +42,15 @@ test-e2e: GO_TEST_PACKAGES :=./test/e2e/...
 test-e2e: GO_TEST_FLAGS += -v
 test-e2e: test-unit
 
+# these are extremely slow serial e2e encryption tests that modify the cluster's global state
+.PHONY: test-e2e-encryption
+test-e2e-encryption: GO_TEST_PACKAGES :=./test/e2e-encryption/...
+test-e2e-encryption: GO_TEST_FLAGS += -v
+test-e2e-encryption: GO_TEST_FLAGS += -timeout 3h
+test-e2e-encryption: GO_TEST_FLAGS += -p 1
+test-e2e-encryption: GO_TEST_FLAGS += -parallel 1
+test-e2e-encryption: test-unit
```

Comment:
how is this picked up? do we need an openshift/release job?

Reply:
https://github.com/openshift/release/pull/5026